### DISCUSSION TITLE
Be able to delete repositories and files on mobile

### DISCUSSION THREAD

**Discussion by Liveeee7219** [community/community#145061] [Mobile]
Being able to delete repositories on mobile would be really helpful for people on the go who need to delete files and repositories

**Comment by SjxSubham** [comment]
True.. and also if we get support to modify/edit codes of our own repos ..that will be great

**Reply by Liveeee7219** [reply]
You already can

**Reply by Liveeee7219** [reply]
By clicking the 3 dots and clicking edit file
